Makefile

```diff
@@ -65,34 +60,33 @@ install-mock-generators:
     go install github.com/vektra/mockery/v2@v2.13.1; \
     go install github.com/golang/mock/mockgen@v1.3.1;
 
-############################################################################################
-
 .PHONY: install-tools
-install-tools: crypto_setup_tests crypto_setup_gopath check-go-version install-mock-generators
+install-tools: crypto_setup_gopath check-go-version install-mock-generators
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ${GOPATH}/bin v1.49.0; \
 	cd ${GOPATH}; \
 	go install github.com/golang/protobuf/protoc-gen-go@v1.3.2; \
 	go install github.com/uber/prototool/cmd/prototool@v1.9.0; \
 	go install github.com/gogo/protobuf/protoc-gen-gofast@latest; \
 	go install golang.org/x/tools/cmd/stringer@master;
 
-.PHONY: unittest
-unittest: unittest-main
-	$(MAKE) -C crypto test
-	$(MAKE) -C integration test
+.PHONY: verify-mocks
+verify-mocks: generate-mocks
+	git diff --exit-code
+
+############################################################################################
 
 .PHONY: emulator-build
 emulator-build:
 	# test the fvm package compiles with Relic library disabled (required for the emulator build)
 	cd ./fvm && go test ./... -run=NoTestHasThisPrefix
 
-.PHONY: emulator-build
+.PHONY: fuzz-fvm
```

Comment:
This target isn't called by any job. Is that expected?

Reply:
Yes - that fuzz job doesn't end and keeps fuzzing until you manually stop it so I think it's meant for local running only. I thought it was an error that it was tagged with the same target name `emulator-build` as the actual emulator building target so I renamed it.